public/app/plugins/panel/stat/AlignmentFactorText.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+import { Input } from '@grafana/ui';
+import { css } from 'emotion';
+import { config } from 'app/core/config';
+
+interface Props {
+  value?: string;
+  onChange: (value?: string) => void;
+}
+
+export const AlignmentFactorTextEditor: React.FC<Props> = ({ value, onChange }) => {
```

Comment:
NOTE:  this was my direct text option, and should be removed if we stick with the `<Select` version